cmd/gctcli/commands.go

```diff
@@ -1243,10 +1245,26 @@ var getOrdersCommand = cli.Command{
 			Name:  "pair",
 			Usage: "the currency pair to get orders for",
 		},
+		cli.StringFlag{
+			Name:        "start",
+			Usage:       "start date, optional. Will filter any results before this date",
+			Value:       time.Now().AddDate(-1, 0, 0).Round(kline.OneDay.Duration()).Format(common.SimpleTimeFormat),
+			Destination: &startTime,
+		},
+		cli.StringFlag{
+			Name:        "end",
+			Usage:       "end date, optional. Will filter any results after this date",
+			Value:       time.Now().Round(kline.OneDay.Duration()).Format(common.SimpleTimeFormat),
```

Comment:
Would be nice to have this default to `time.Now().Format(common.SimpleTimeFormat)` for extra convenience to the user